ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -106,6 +106,10 @@ func clusterUpFlags() []cli.Flag {
 			Usage: "[Optional] Specify the AMI ID for your container instances. Defaults to amazon-ecs-optimized AMI.",
 		},
 		cli.BoolFlag{
+			Name:  command.NoAutoAssignPublicIPAddress,
+			Usage: "[Optional] Do not assign public IP addresses to new instances in this VPC. Unless this option is specified, new instances in this VPC receive an automatically assigned public IP address.",
```

Comment:
Nitpick: I know its stated in the text. But do you think it would help if you had an explicit "Defaults to false" at the end.